bundles/core/src/test/java/com/adobe/cq/commerce/core/components/internal/services/ComponentsConfigurationAdapterFactoryTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ *  Copyright 2020 Adobe. All rights reserved.
```

Comment:
Sorry to be picky, this header is not formatted like all the other headers we use.